devicetypes/smartthings/testing/simulated-thermostat.src/simulated-thermostat.groovy

```diff
@@ -306,6 +314,8 @@ def parse(String description) {
 
 def ping() {
     log.trace "Executing ping"
+    sendEvent(name: "DeviceWatch-DeviceStatus", value: "online")
```

Comment:
if this is untracked, i don't think we need the ping() method, but check with the DH team